microsoft-edge/devtools-guide-chromium/whats-new/2024/11/devtools-131.md

```diff
@@ -0,0 +1,82 @@
+---
+title: What's New in DevTools (Microsoft Edge 131)
+description: "Can disable Edge experiment settings. View CSS selector stats information in Insights tab of sidebar panel in Performance tool. Fixed edit field to exit after pressing tab after auto-completion. Removed Visual Studio Code themes. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 11/14/2024
+---
+# What's New in DevTools (Microsoft Edge 131)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Visual Studio Code themes are removed
+
+<!-- Subtitle: The Visual Studio Code themes are removed and themes will revert to the default themes: Light+ or Dark+.-->
+
+The Visual Studio Code themes feature is now deprecated.  These themes will revert to the **System preference** theme by default.  The **Light+** and **Dark+** themes continue to be supported.
+
+
+<!-- ====================================================================== -->
+## Edge experiment settings can be disabled
+
+<!-- Go to Settings > Experiments to enable or disable any experiments you choose. -->
+
+Previously, the following experiments that were enabled by default couldn't be disabled, even after clearing their checkboxes in the **Settings > Experiments** page and reloading DevTools:
+* **Enable webhint**
+* **Show issues in Elements**
+* **Open source files in Visual Studio Code**
+
+This is now fixed, and you can disable these experiments.
+
+
+<!-- ====================================================================== -->
+## View CSS selector stats information in the Insights tab of the sidebar panel in the Performance tool
+
+<!-- When the "Enable CSS selector stats (slow)" setting is turned on, view selector stats information in the left sidebar of the Performance tool.-->
+
+The left sidebar in the **Performance** tool includes information about CSS selector costs in the **Insights** tab.  To view this, select the **Enable CSS selector stats (slow)** checkbox, take a performance recording, open the sidebar, and then expand the **CSS selector costs** section:
+
+![CSS selector costs info](./devtools-131-images/selector-stats-info.png)
```

Comment:
in Beta, for CSS selector costs UI, in Beta, i'm getting the addl orange-boxed content, should i redo png to include that?  @leahmsft 

![image](https://github.com/user-attachments/assets/21c33ad7-2c9f-4933-b44f-40c5e1fdd2c9)

Reply:
added